indexer/src/main/java/au/org/aodn/esindexer/utils/GeometryUtils.java

```diff
@@ -75,7 +88,7 @@ public static void init() {
                     Geometry simplifiedGeometry = DouglasPeuckerSimplifier
                             .simplify(landFeatureGeometry, getCoastalPrecision()); // Adjust tolerance
 
-                    geometries.add(simplifiedGeometry);
+                    geometries.add(reducer != null ? reducer.reduce(simplifiedGeometry) : simplifiedGeometry);
```

Comment:
The reducer is a function from locationtech, here we try to simplify the land geo location so that we generate simpler spatial extents with land stripped. Therefore the result of the reduce will be a polygon describing the land on earth. Not necessary not testable but the expected result will be super big. I can add one test to verify a polygon, but not sure how useful it will be

Reply:
added a test case